models/DataObject/ClassDefinition/Data/UrlSlug.php

```diff
@@ -276,8 +276,8 @@ public function save($object, $params = [])
                             }
 
                             // if now exception is thrown then the slug is owned by a diffrent object/field
-                            throw new \Exception('Unique constraint violated. Slug alreay used by object '
-                                . $existingSlug->getFieldname() . ', fieldname: ' . $existingSlug->getFieldname());
+                            throw new \Exception('Unique constraint violated. Slug already used by object '
```

Comment:
what about adding also the slug in the output?
```suggestion
                            throw new \Exception('Unique constraint violated. Slug "' . $slug['slug'] . '" already used by object '
```